### DISCUSSION TITLE
To generally talk about the project without creating unnecessary issues

### DISCUSSION THREAD

**Discussion by mohitgedar** [kunjgit/GameZone#4146] [General]
I have come across some things that would make this project great to work on and would like to discuss about it with community and the maintainers .

**Comment by mohitgedar** [comment]
## There is a small problem that I feel can be addressed 
 ### People have submitted games, so many games , great games . 
 - problem is the codes are not commented at all in majority , so it makes really hard to work on them . 
 - second some games have used some build tool to create a running game , i don't know what the alternative can be , but the build tool make the code so compact and makes it kind of encrypted therefore we can't work on those also to improve them or fix bugs

what are your thoughts ?
- I don't know about games that use build tools , but the simple games will be very easy to work on , if they are well commented

**Reply by SuperGamer001** [reply]
If this issue hasn't been resolved, I have a few recommendations for ya! 😁

- For the comments, you can make it a rule to always comment their code in order to submit.
- For the web tools, maybe the root can have an exec tool so the other games can use it to install needed npm tools and libraries.

I'd like the freedom of trying to allow as much creativity as possible (with secure tools). I know it has been awhile after this was posted, but still good to bring up.
